browser/src/react-redux/App/SignupPage.jsx

```diff
@@ -1,8 +1,8 @@
-import React, { Component } from 'react';
```

Comment:
It looks like all of these changes are just from your linter. As I mentioned below, let's go ahead and reverse this. Typically, we want keep the PR as trim as we can, with only changes committed that are relevant to the feature or bug that you're working on.